📝(content) Content: Bluetooth microphone super resolution

jkcinouye opened the pull request:
Changes to be committed:
	modified:   site/_data/contributors.js
	new file:   site/en/posts/bluetooth-microphone-super-resolution-better-nbs-sound-quality-through-machine-learning.md

jkcinouye committed:
Add content: Bluetooth Microphone Super Resolution

 Changes to be committed:
	modified:   site/_data/contributors.js
	new file:   site/en/posts/bluetooth-microphone-super-resolution-better-nbs-sound-quality-through-machine-learning.md

Snugug commented on site/en/posts/bluetooth-microphone-super-resolution-better-nbs-sound-quality-through-machine-learning.md:
```diff
@@ -0,0 +1,110 @@
+
+Listen to the below before and after demo samples processed by the model.
+
+<table>
+  <thead>
+    <tr>
+      <th>Audio signal from Bluetooth NBS connections</th>
+      <th>Audio signal processed by the model</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <figure>
+            <a href="https://firebasestorage.googleapis.com/v0/b/cros-staging.appspot.com/o/posts%2Fbluetooth-nbs%2Fnbs_input_4_airpods_pro.wav?alt=media">nbs_input_4_airpods_pro.wav</a>
+            <img src="insert_image_url_here" alt="A spectrogram representing the unprocessed audio clip. The scale measures from 0 to 12 kHz, with heat colors presenting a spectrum from -80 dB to +0 dB. The spectrogram shows that the audio clip does not exceed 4 kHz.">
+            <figcaption>From the spectrogram, we can see that the highest frequency is capped at 4 kHz.</figcaption>
+        </figure>
+        </td>
+        <td>
+        <figure>
+            <a href="https://firebasestorage.googleapis.com/v0/b/cros-staging.appspot.com/o/posts%2Fbluetooth-nbs%2Fnbs_processed_4_airpods_pro.wav?alt=media">nbs_processed_airpods_pro.wav</a>
+        <img src="insert_image_url_here" alt="A spectrogram representing the processed audio clip. The scale measures from 0 to 12 kHz, with heat colors presenting a spectrum from -80 dB to +0 dB. The spectrogram shows that the audio clip now peaks up to 12 kHz.">
+        <figcaption>The highest frequency becomes 12 kHz and some of the higher frequency components are reconstructed.</figcaption>
+        </figure>
+    </td>
+    </tr>
+  </tbody>
+</table>
```
We don't want to do this as a table. Please make these using our standard table Markdown.

jkcinouye committed:
Update bluetooth-microphone-super-resolution-better-nbs-sound-quality-through-machine-learning.md

Added images.

jkcinouye committed:
Changes to be committed:
	modified:   site/en/posts/bluetooth-microphone-super-resolution-better-nbs-sound-quality-through-machine-learning.md

Snugug commented on site/en/posts/bluetooth-microphone-super-resolution-better-nbs-sound-quality-through-machine-learning.md:
```diff
@@ -0,0 +1,87 @@
+
+| Audio signal from Bluetooth NBS connections                                                                                | Audio signal processed by the model                                                                                                            |
+| -------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
+| #1[From the spectrogram, we can see that the highest frequency is capped at 4 kHz.](ix://posts/bluetooth-nbs/Inline-2.png) | #2[The highest frequency becomes 12 kHz and some of the higher frequency components are reconstructed.](ix://posts/bluetooth-nbs/Inline-3.png) |
```
If we're going to do these as a table, they don't need to be figures, and the links to download the audio files have been lost in this conversion.

jkcinouye committed:
Changes to be committed:
	modified:   site/en/posts/bluetooth-microphone-super-resolution-better-nbs-sound-quality-through-machine-learning.md

Update to tables.

jkcinouye committed:
Changes to be committed:
	modified:   site/en/posts/bluetooth-microphone-super-resolution-better-nbs-sound-quality-through-machine-learning.md

Table spacing.
